includes/class-wc-payments-utils.php

```diff
@@ -875,6 +875,25 @@ public static function is_any_bnpl_method_available( array $enabled_methods, str
 		return false;
 	}
 
+	/**
+	 * Check if any BNPL method is available for a given country.
+	 *
+	 * @param array  $enabled_methods Array of enabled BNPL methods.
+	 * @param string $country_code Country code.
+	 * @param string $currency_code Currency code.
+	 * @return bool True if any BNPL method is available, false otherwise.
+	 */
+	public static function is_any_bnpl_supporting_country( array $enabled_methods, string $country_code, string $currency_code ): bool {
```

Comment:
It might seem at first glance that this function should be reusing components similar to `is_any_bnpl_method_available`, but after trying a bit, it doesn't seem to simplify things. So I kept it separate.